WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -188,29 +188,6 @@ class CommentDetailViewController: UIViewController, NoResultsViewHost {
         return appearance
     }()
 
-    // MARK: Nav Bar Buttons
-
-    private(set) lazy var editBarButtonItem: UIBarButtonItem = {
-        let button = UIBarButtonItem(barButtonSystemItem: .edit,
-                               target: self,
-                               action: #selector(editButtonTapped))
-        button.accessibilityLabel = NSLocalizedString("Edit comment", comment: "Accessibility label for button to edit a comment from a notification")
-        return button
-    }()
-
-    private(set) lazy var shareBarButtonItem: UIBarButtonItem = {
-        let button = UIBarButtonItem(
-            image: comment.allowsModeration()
-            ? UIImage(systemName: Style.Content.ellipsisIconImageName)
-            : UIImage(systemName: Style.Content.shareIconImageName),
-            style: .plain,
-            target: self,
-            action: #selector(shareCommentURL)
-        )
-        button.accessibilityLabel = NSLocalizedString("Share comment", comment: "Accessibility label for button to share a comment from a notification")
-        return button
-    }()
```

Comment:
Edit and share bar buttons were removed from the navigation bar, because now they're buttons in the comment's menu.